pyproject.toml

```diff
@@ -22,3 +22,81 @@ environment = 'RUSTUP_TOOLCHAIN="stable"'
 [tool.cibuildwheel.linux]
 before-all = "yum install -y wget && {package}/tools/install_rust.sh"
 environment = 'PATH="$PATH:$HOME/.cargo/bin" CARGO_NET_GIT_FETCH_WITH_CLI="true" RUSTUP_TOOLCHAIN="stable"'
+
+[tool.pylint.main]
+extension-pkg-allow-list = ["retworkx", "numpy", "tweedledum", "qiskit._accelerate", "rustworkx"]
+load-plugins = ["pylint.extensions.docparams", "pylint.extensions.docstyle"]
+py-version = "3.7"  # update it when bumping minimum supported python version
```

Comment:
Good call. Thank you!

Reply:
Unfortunately it's not enough to completely avoid the python version dependence of the test_transforms lint suppression.